ch1-basic/ch1-03-array-string-and-slice.md

```diff
@@ -151,7 +151,7 @@ Cấu trúc [reflect.StringHeader](https://golang.org/src/reflect/value.go?s=565
 
 ```go
 type StringHeader struct {
-    // con trỏ địa chỉ vùng nhớ string
+    // chứa giá trị con trỏ tới array của string
```

Comment:
nit: "chứa đỉa chỉ trong bộ nhớ của con trỏ trỏ tới underlying array của string"

Reply:
It is clear. Tks bro.